manual/features.tex

```diff
@@ -1,38 +1,38 @@
 \chapter{Features of QMCPACK}
 \label{chap:features}
 \section{Production-level features}
-The following lists the main production level features of QMCPACK. If
-you do not see a specific feature that you are interested in listed,
-see the remainder of this manual and ask to see if specific feature is
-available or can be made full production level quickly.
+The following list contains the main production-level features of QMCPACK. If
+you do not see a specific feature that you are interested in,
+see the remainder of this manual and ask whether that specific feature is
+available or can be quickly  brought to the full production level.
 
 \begin{itemize}
-\item Variational Monte Carlo
-\item Diffusion Monte Carlo
+\item Variational Monte Carlo (VMC)
+\item Diffusion Monte Carlo (DMC)
 \item Reptation Monte Carlo
-\item Single and multi-determinant Slater Jastrow wavefunctions
-\item Wavefunction updates using optimized multi-determinant algorithm of Clark et al.
+\item Single and multideterminant Slater Jastrow wavefunctions
+\item Wavefunction updates using optimized multideterminant algorithm of Clark et al.
 \item Backflow wavefunctions
 \item One, two, and three-body Jastrow factors
 \item Excited state calculations via flexible occupancy assignment of Slater determinants
-\item All electron and non-local pseudopotential calculations
-\item Casula T-moves for variational evaluation of non-local
-  pseudopotentials (non-size consistent and size-consistent variants)
+\item All electron and nonlocal pseudopotential calculations
+\item Casula T-moves for variational evaluation of nonlocal
+  pseudopotentials (nonsize consistent and size-consistent variants)
```

Comment:
I don't think 'nonsize' is a word, and 'size' should be more connected to 'consistent' the preceeding 'non'.   Should this be 'non-size-consistent'?

Reply:
Fixed